p2pserver/actor/server/common.go

```diff
@@ -132,3 +132,15 @@ type TransmitConsensusMsgReq struct {
 	Target uint64
 	Msg    ptypes.Message
 }
+
+type GetNbrPeerVersionInfosReq struct {
+}
+
+type NbrPeerVersionInfo struct {
+	ID      uint64
+	Version uint32
```

Comment:
I think what we need is the node's release version, not p2p protocal version.

Reply:
Ok, have changed to node's release version.